imap_processing/swapi/l1/swapi_l1.py

```diff
@@ -138,11 +139,12 @@ def find_sweep_starts(packets: xr.Dataset) -> np.ndarray:
 
     Returns
     -------
-    numpy.ndarray
+    indices_start : numpy.ndarray
         Array of indices of start cycle.
     """
     if packets["epoch"].size < 12:
-        return np.array([], np.int64)
+        indices_start = np.array([], np.int64)
+        return indices_start
```

Comment:
I think this can be reverted.
```suggestion
        return np.array([], np.int64)
```